pkg/operator/api/userconfig/apis.go

```diff
@@ -80,12 +100,23 @@ func (apis APIs) Validate() error {
 }
 
 func (api *API) Validate() error {
-	if api.ModelPath == nil && api.Model == nil {
-		return errors.Wrap(ErrorSpecifyOnlyOneMissing("model_name", "model_path"), Identify(api))
+	if api.ExternalModel == nil && api.Model == nil {
+		return errors.Wrap(ErrorSpecifyOnlyOneMissing("model", "external_model"), Identify(api))
```

Comment:
can you use the consts in `config_key.go` for the field names here and below? You might need to add one for external_model